setup.py

```diff
@@ -2,9 +2,10 @@
 
 from setuptools import find_packages, setup
 
-# TODO: we can make this a conda package and have but that will be on the next iteration.
-# TODO: Some more work can be done on the setup requires but for now use the conda_vendor_test.yml
-# if you want to build the wheel.
+# TODO: we can make this a conda package and have but that will be on the next
```

Comment:
we can make this a conda package and have ... ?

Reply:
not my comment, leaving for now